packages/vendure-plugin-invoices/src/util/order-calculations.ts

```diff
@@ -9,16 +8,18 @@ import { InvoiceOrderTotals } from '../entities/invoice.entity';
 export function reverseOrderTotals(
   orderTotal: InvoiceOrderTotals
 ): InvoiceOrderTotals {
-  const summaries = orderTotal.taxSummaries.map((summary) => {
+  const summaries = orderTotal.taxSummaries?.map((summary) => {
     return {
       ...summary,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
       taxBase: -summary.taxBase,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
       taxTotal: -summary.taxTotal,
-    };
+    } as InvoiceOrderTotals;
```

Comment:
Hmm whats going on here? Why do we need to assert as `InvoiceOrderTotals`? 


Ahh ok, so you are now using the generated type probably? Maybe you can create a Graphql type for the tax summaries also, instead of a scalar, so that we have correct typing for this.

Reply:
`OrderTaxSummary` already exists in vendure, so when try to declare it in this plugin, vendure complains.

Reply:
Ahh yes, I see. Leave it for now, I will check it out locally. 

I think the best thing to do, is to create a new Graphl type `InvoiceTaxSummary`. It happens to be the same as OrderTaxSummary, but they have no relation to eachother, so they should actually be different types. I was lazy, so I originally just reused the OrderTaxSummary :upside_down_face: 

That should solve it. We can then also get rid of the InvoiceOrderTotals in the invoice entity, and only use the generated type

Reply:
@dalyathan I think this fixed most of the issues: https://github.com/Pinelab-studio/pinelab-vendure-plugins/pull/488/commits/134c80ba45d635f8519ecd1ddb5454b0f49d558b